src/posit/connect/resources.py

```diff
@@ -62,7 +69,11 @@ def __init__(self, ctx: Context, path: str, /, **attributes):
         self._path = path
 
 
-class _Resource(dict):
+class Resource(Protocol):
+    def __getitem__(self, key: Hashable) -> Any: ...
+
+
+class _Resource(dict, Resource):
```

Comment:
This class is no longer needed?

Reply:
Correct. The BaseResource class should go away. But that likely requires finishing the migration to the new implicit subtyping pattern. Which I am planning to complete.